dds_web/api/api_s3_connector.py

```diff
@@ -126,3 +126,25 @@ def remove_one(self, file, *args, **kwargs):
             removed = True
 
         return removed, error
+
+    def generate_get_url(self, key):
+        """Generate presigned urls for get requests."""
+
+        url = self.resource.meta.client.generate_presigned_url(
```

Comment:
Would be nice to add a comment here about the question we had regarding http requests made or not. The conclusion was that it probably doesn't make requests I think.

Reply:
Yeah, it doesn't. And sure, I can add a comment about it.